backend/controllers/iepDetails.js

```diff
@@ -74,8 +74,55 @@ const getIepDetailsFactory = elite2Api => {
     }
   }
 
+  const iconForDifference = {
+    '1': 'Green_arrow.png',
+    '2': 'Double_green_arrow.png',
+    '3': 'TripleGreenArrow.png',
+    '-1': 'Red_arrow.png',
+    '-2': 'Double_red_arrow.png',
+  }
+
+  const levelToIntMap = {
+    Entry: 1,
+    Basic: 2,
+    Standard: 3,
+    Enhanced: 4,
+  }
+
+  const getPossibleLevels = async (context, currentIepLevel, agencyId) => {
+    const levels = await elite2Api.getAgencyIepLevels(context, agencyId)
+
+    return levels
+      .filter(level => level.iepDescription !== currentIepLevel)
+      .filter(level => level.iepDescription !== 'Entry')
+      .map(level => {
+        let diff
+        if (levelToIntMap[level.iepDescription] && levelToIntMap[currentIepLevel]) {
+          diff = levelToIntMap[level.iepDescription] - levelToIntMap[currentIepLevel]
+        } else {
+          // This is a custom level for which we do not have an icon. Always show it last.
+          diff = 1000
+        }
+        return {
+          title: level.iepDescription,
+          value: level.iepLevel,
+          image: iconForDifference[diff.toString()] || '',
+          delta: Math.abs(diff), // Used for ordering so the biggest change is shown last regardless of up or down.
+          diff,
```

Comment:
Would it make sense to just rename this to delta?

Reply:
The diff is used to determine which icon to use and the delta is used for ordering, they serve different purposes.

Reply:
Have removed delta, we just return levelDifference as part of this as it's used for sorting.